ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -73,6 +78,41 @@ export default class StatusDropdown extends React.Component {
     };
   };
 
+  getNumberOfNonWorkingDays(createdAt, dueDate) {
+    let count = 0;
```

Comment:
`count` is too general, should be replaced with something more descriptive. Just the name `count` leaves a question of what we are counting here.